### ISSUE TITLE
CLI support for pallets

### ISSUE THREAD

**Issue by ltfschoen** [sunshine-protocol/sunshine-bounty#148]
When I reviewed the CLI it doesn't appear to yet support functionality from the following pallets Bounty Utils, Grant, Treasury, Drip, or Court. What's the plan here?

**Comment by 4meta5** [comment]
Yes, this is an open issue. It could be added for the modules listed in particular. I'm planning to do a refactor on the org module at some point as mentioned in #134 so it might be better to wait until after that refactor is done because changes will percolate up.
